content/docs/19-kubernetes-school/00-how-to/00-how-to.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "How to"
+metaTitle: "Get started with a quick Kubernetes How-to"
+metaDescription: "Kubernetes School with How to"
+icon: "laptop"
+hideToC: false
+fullWidth: false
+hiddenFromNav: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# How To
+
+Learn about core Kubernetes concepts and how you can apply them on Spectro Cloud Palette. <p></p><br />
```

Comment:
The HTML at the end is not needed. Let's remove it as we are trying to avoid using HTML in the markdown.